src/components/LoginSignUp/SignUp.js

```diff
@@ -0,0 +1,56 @@
+// @flow strict-local
+
+import React, { useState } from "react";
+import { Button, Text, TextInput, View } from "react-native";
+import type { Node } from "react";
+
+import { viewStyles, textStyles } from "../../styles/login/login";
+import AuthenticationService from "./AuthenticationService";
+
+const SignUp = (): Node => {
+  const [email, setEmail] = useState( "" );
+  const [username, setUsername] = useState( "" );
+  const [password, setPassword] = useState( "" );
+
+  const register = async () => {
+    const error = await AuthenticationService.registerUser(
+      email,
+      username,
+      password
+    );
+
+    console.log( "Register", error );
```

Comment:
It might be a good idea to either clean up these console.logs or at least add the [babel-plugin-transform-remove-console](https://babeljs.io/docs/en/babel-plugin-transform-remove-console/) package to dev dependencies so they will be stripped out of production builds.

Reply:
In general, we have similar `console.log` in the AuthenticationService module - but we do need those for debugging purposes, when a user sends us logs. What logging library do you use in Seek?